core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -114,14 +114,11 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setCategory = async function(category) {
-    await action.click('Neutral element', neutralElement);
-    await action.waitForAutosave();
```

Comment:
Why remove this?

Reply:
@kevintab95 FYI SAEb-ai documented their reasoning in a [debugging doc](https://docs.google.com/document/d/1CxS7Vhc6JWVHTh6Mgt9gMbYEbRNfj60fpHkjVwrD-9M/edit?usp=sharing)

Reply:
Hi @kevintab95!
I was debugging to fix the flake ****

Reply:
Now regarding the **waitForAutoSave**,
I coundn't find even find any saving element coming up aftea category is selected or a dropdown is clicked to open it.
So U8NWXD and I thought to better remove it.

Reply:
Thanks left a comment on the doc. PTAL!

Reply:
Thanks left a comment on the doc. PTAL!

Reply:
> I coundn't find even find any saving element coming up after category is selected or a dropdown is clicked to open it.


This is not true I think. Please see GIF

![auto-saving](https://user-images.githubusercontent.com/11008603/116825814-788b8200-abae-11eb-9af0-df2cdd3e2b2a.gif)

Reply:
I don't know why I couldn't get this on my local setup but got this on the test server. Hence, added it again!

Reply:
Hi @SAEb-ai has this been addressed?